cmd/loader.go

```diff
@@ -188,3 +198,17 @@ func runTraceMode(cfg *config.LoaderConfiguration, iatOnly bool, generated bool)
 
 	experimentDriver.RunExperiment(iatOnly, generated)
 }
+
+func getTrace() bool {
+	_, err := exec.Command("git", "lfs", "pull").CombinedOutput()
+	if err != nil {
+		log.Warnf("Failed to get traces")
+		return false
+	}
+	_, err = exec.Command("tar", "-xzf", "data/traces/reference/sampled_150.tar.gz", "-C", "data/traces").CombinedOutput()
+	if err != nil {
+		log.Warnf("Failed to extract sample")
+		return false
+	}
+	return true
+}
```

Comment:
I don't think this should be git and tar commands should be part of the the loader. The user should be responsible for making sure these files exist and are in proper format.